### PR TITLE
Update CMakeLists.txt

### PR DESCRIPTION
**Pull request by juugcatm**
Installed xmr-stak on fresh Ubuntu 16.04 with just AMDGPU-PRO and required these changes and use of OpenCL_ROOT=/opt/amdgpu-pro for successful compilation with OpenCL. Also requires the opencl-headers package from Ubuntu's repository.

Please make sure your PR is against **dev** branch. Merging PRs directly into master branch would interfere with our workflow.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -228,6 +228,7 @@ if(OpenCL_ENABLE)
             ENV AMDAPPSDKROOT
             ENV ATISTREAMSDKROOT
             ENV "PROGRAMFILES(X86)"
+	    /usr/include
```
Comment (psychocrypt) [inline CMakeLists.txt]: please fix the indention and add only `/usr` because `include` is selected by the suffix

File: CMakeLists.txt [hunk 2]
```diff
@@ -246,6 +247,7 @@ if(OpenCL_ENABLE)
         PATH_SUFFIXES
             "AMD APP/lib/x86_64"
             lib/x86_64
+	    lib/x86_64-linux-gnu
```
Comment (psychocrypt) [inline CMakeLists.txt]: please fix the indention (spaces only in this file)

File: CMakeLists.txt (hunk 1 above)
Comment (juugcatm) [reply CMakeLists.txt]: Updated.

File: CMakeLists.txt (hunk 2 above)
Comment (juugcatm) [reply CMakeLists.txt]: Updated.

**Comment by psychocrypt** [comment]
thx for the update and the pull request

**Pull request merged**